nodejs/.env-example

```diff
@@ -20,7 +20,7 @@ MAINNET_OPERATOR_PRIVATE_KEY=
 TESTNET_ACCOUNT_ID=0.0.13365721
 TESTNET_ACCOUNT_ID_EVM=0x9132b50e8dd60c1a61f1d79f8d90669d690ebf53
 TESTNET_OPERATOR_PRIVATE_KEY=3030020100300706052b8104000a04220420c63c8cbe79f519f77789f26a8a6709feb1c2f7a5cc123f4b0593f50880a6dbc8
-
+TESTNET_OPERATOR_PRIVATE_EVM_KEY=2651c08778a314549a2b65c81b9ff1dc4871cbd13a04f9a722cc3d00d083b248
```

Comment:
This is TESNET right? (friendly reminder this repo is public)

Reply:
yes testnet 😎